src/sentry/static/sentry/app/components/assistant/getGuidesContent.tsx

```diff
@@ -210,3 +214,102 @@ export default function getGuidesContent(orgSlug: string | null): GuidesContent
     },
   ];
 }
+
+function getDemoModeGuides(): GuidesContent {
+  return [
+    {
+      guide: 'sidebar',
+      requiredTargets: ['projects', 'issues'],
+      priority: 2,
```

Comment:
would each guide's priority need to be changed when adding a new guide? could make sense to reverse the priority order so 0 or 1 is the highest.

Reply:
@maheskett only if the other guides have a priority (if no priority is given, then it's the lowest priority). I can definitely flip the priority so higher lower is a higher priority if people think that makes sense. I'd be curious what other people think.